sopel/modules/url.py

```diff
@@ -98,6 +98,14 @@ def setup(bot):
         bot.memory['shortened_urls'] = tools.SopelMemory()
 
 
+def shutdown(bot):
+    for key in ['url_exclude', 'last_seen_url']:
```

Comment:
Should have mentioned yesterday: Leaving a comment here explaining (briefly) why `shortened_urls` isn't cleared would help ensure that it's not re-added later without good reason.

Reply:
:+1: Done.